pkg/storaged/filesystem/utils.jsx

```diff
@@ -35,21 +35,19 @@ import { mounting_dialog } from "./mounting-dialog.jsx";
 
 const _ = cockpit.gettext;
 
-export function is_mounted(client, block) {
+/* is mounted with an entry in fstab */
+export function is_mounted(client, block, subvol) {
```

Comment:
What about passing fstab as an argument? We could do this maybe as a general principle in the code: replace most calls to get_fstab_config with a fstab property that is passed into the components. And the only call to get_fstab_config would be in make_block_page... hmm.

Reply:
Thought about that before, but that sounds like follow up territory.